docs/modules/automation-testing/pages/scripting/create-image-injection-script.adoc

```diff
@@ -0,0 +1,98 @@
+= Create image injection script
+:navtitle: Create an image injection script
+

```

Comment:
Image injection is likely just one part of a script. What about "Add image injection to Appium script" cc: @tungmhoang (I know it is not consistent with biometrics - we would want to change that later)

Reply:
updated.